docs/install.md

```diff
@@ -55,9 +56,16 @@ $ kubectl gadget version
 ```
 
 ### Compile from source
+#### Build requirements
 
-To build Inspektor Gadget from source, you'll need to have a Golang version
-1.19 or higher installed.
+To build Inspektor Gadget from source, the following requirements need to be installed and configured on your system:
+- Golang version 1.19 or higher
+- Docker Engine
+- gcc compiler
+- make
+
+#### Compile
+For compiling, clone this repository within the following directory structure: `go/src/github.com/inspektor-gadget/`:
```

Comment:
I suggest adding the explanation in commands:
```
mkdir -p ~/go/src/github.com/inspektor-gadget/
cd ~/go/src/github.com/inspektor-gadget/
git clone ...
```

Reply:
I also like this idea better. Added ✅